src/ml-engine/tests/test_data_classes.py

```
@@ -10,34 +10,8 @@ def sample_json():
-              "Date": "2025-05-03T15:30:00.123Z",
-              "TVL": 123,
-              "APY": 0.0523,
-              "Base APY": 0.05,
-              "Reward APY": 0.0023
-            }
-          ]
-        }
-      ]
-    }
-    """
+    with open(os.path.join(os.path.dirname(__file__), "test_data/black_litterman_explicit_view_test_data.json"), "r") as file:
```

Ruff (0.11.9)

Undefined name `os`

(F821)